docs/features/star-search.md

```diff
@@ -23,6 +28,30 @@ keywords:
 - **Hidden Experts**: Discover untapped talent in the ecosystem.
 
 ![StarSearch](../../static/img/star-search.png)
-## How to Access StarSearch
 
-Currently, StarSearch is in development, but you can [sign up for the waitlist now](https://app.opensauced.pizza/star-search/waitlist).
\ No newline at end of file
+### Queries Suggestions
+
+#### **Contributor Activities**
+    - Show me the contributions of \{username}
+    - What \{technology} contributions has \{username} made recently?
+    - What issues has \{username} commented on?
+- **Work-Based Connections**: 
+    - Who has contributed to both project A and project B?
+    - Who are the contributors to both \{technology} and \{technology}?
+- **Hidden Experts**: 
+    - Who are the experts in \{technology}?
+    - Who are the most prevalent contributors to the \{technology} ecosystem?
+
+#### Technology Insights
+    - What are the most popular projects in the \{technology} ecosystem? 
+    - When was the last release of \{project}?
+    - Whare are the breaking changes in \{project}'s last release?
+
+### FAQs
+- **Can you continue conversations with StarSearch?**
+    - Not yet, but stay tuned!
+- **Can you access your search history?**
+    - Not yet.
```

Comment:
**Question**: As FAQ's are usually questions from individuals to us, would it be better to use "I" as a subject here just like our [FAQ's page](https://docs.opensauced.pizza/welcome/faqs/)?

 ```suggestion
- **Can I continue conversations with StarSearch?**
    - Not yet, but stay tuned!
- **Can I access my search history?**
    - Not yet.
```